package.json

```diff
@@ -4,6 +4,7 @@
   "version": "0.0.0",
   "type": "module",
   "scripts": {
+    "start": "react-scripts start",
```

Comment:
You will have to omit react-scripts in a vite project. Please refer to this [link](https://cathalmacdonnacha.com/migrating-from-create-react-app-cra-to-vite) in section 8 for your reference.